app/components/emoji/emoji.js

```diff
@@ -15,20 +15,23 @@ export default class Emoji extends React.PureComponent {
         customEmojis: PropTypes.object,
         emojiName: PropTypes.string.isRequired,
         literal: PropTypes.string,
+        padding: PropTypes.number,
```

Comment:
I was having an issue setting the size of the emoji outside of modifying the padding on the actual emoji component which is why I decided to created a prop. No matter what I set the size to the emoji wouldn't resize unless the padding was changed. I can play with it some more if we want to avoid it, but I wasn't having a lot of luck.